ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/ParallelIndexerFactory.scala

```diff
@@ -108,12 +127,11 @@ object ParallelIndexerFactory {
                 .map(_ -> System.nanoTime())
             )
               .map(_ => ())
-              .keepAlive(
+              .keepAlive( // TODO ha: remove as stable. This keepAlive approach was introduced for safety with async commit. This is still needed until HA is mandatory for Postgres to ensure safety with async commit.
```

Comment:
What removes the need of this keepAlive here? is it the `PollingChecker` from the `HaCoordinator`? 
If yes, it might be good to mention it here

Reply:
The keepAlive's main purpose here is to let the indexing die if connectivity to DB is lost, to prevent the corner case scenario with async commit, and unnoticed DB restarts.
With postgres+async commit+ha DB restarts cannot go unnoticed, because if the main connection (which holds the main indexer lock) breaks, we loose the indexer lock, and shut down indexing as soon as detected.
(Also we cannot initialize an additional connection to database, if we cannot verify that the main connection still holds the lock.)
I'll make a note about this in the TODO